src/chainparams.cpp

```diff
@@ -325,6 +335,9 @@ class CRegTestParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].bit = 0;
         consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nStartTime = 0;
         consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nTimeout = 999999999999ULL;
+        consensus.vDeployments[Consensus::DEPLOYMENT_BLOCKFINAL].bit = 1;
+        consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nStartTime = 1199145601; // January 1, 2008
+        consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nTimeout = 1230767999; // December 31, 2008
```

Comment:
In ec8fe58bc5e998b68676384327ade069b7a0c321 

Should be `DEPLOYMENT_BLOCKFINAL` not `DEPLOYMENT_LOCKTIME`. Are you sure those dates are reasonable? If you want this to always be on from start, consider instead using `Consensus::BIP9Deployment::ALWAYS_ACTIVE` for `nStartTime` and `Consensus::BIP9Deployment::NO_TIMEOUT` for `nTimeout`.

Reply:
I kept the above comments, but realized in future commits that you already fixed this. Should probably rebase-edit the commit to remove the 2nd-3rd lines entirely here.